### PR TITLE
tr(structure): rework doc structure for release 2.0

### PR DESCRIPTION
**Pull request by tnguyen1**
A global rework of BCD documentation in preparation to 2.0 release.

### CODE REVIEW

File: md/aws_prerequisites.md [hunk 1]
```diff
@@ -138,7 +142,7 @@ Therefore a [default VPC and default subnets](http://docs.aws.amazon.com/AmazonV
     - Download the `.pem` private key file
     - Ensure private key file permissions: `$ chmod 400 <keyfile_basename>.pem`
 
-Bonita Provisioning solution uses official Ubuntu cloud images as EC2 AMI (Amazon Machine Images).
+Bonita Provisioning solution uses official Ubuntu cloud images as EC2 AMI (Amazon Machine Images).
```
Comment (alarreine) [inline md/aws_prerequisites.md]: BCD use == only use OR only support?

File: md/bcd_cli.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+Running the `build` subcommand of the `livingapp` command with `BCD_SCENARIO` environment variable and subcommand `-p` and `-e` options:
+```
+$ export BCD_SCENARIO=scenarios/euwest1_performance.yml
+$ bcd livingapp build -p path/to/livingapp-repo -e Qualification
+```
+
+
+## Multi command chaining
+
+The BCD CLI allows to invoke more than one command in one go. This is useful to chain commands as a pipeline.
```
Comment (alarreine) [inline md/bcd_cli.md:46]: Is it better?
The BCD CLI allows invoking more..
or
The BCD CLI allows you to invoke more..

The verb invoke is usually in the gerund form when following the word allows.

File: md/bcd_cli.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
+# BCD command-line interface
+
+On this page you will find detailed information about the BCD CLI with usage examples and reference.
+
+
+## Requirements
+
+* First the `bcd` command requires that the **`BCD_HOME`** environment variable is set.
```
Comment (alarreine) [inline md/bcd_cli.md]: Is it better?
First, the `bcd` command

File: md/bcd_cli.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
+# BCD command-line interface
+
+On this page you will find detailed information about the BCD CLI with usage examples and reference.
+
+
+## Requirements
+
+* First the `bcd` command requires that the **`BCD_HOME`** environment variable is set.  
+By default `BCD_HOME` is set to `/home/bonita/bonita-continuous-delivery` in the controller container.
+* Next the `bcd` command takes a mandatory option called `scenario` through the `-s / --scenario` command line option.
```
Comment (alarreine) [inline md/bcd_cli.md]: Is it better?
Next, the `bcd` command

File: md/bcd_cli.md [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+
+
+## Requirements
+
+* First the `bcd` command requires that the **`BCD_HOME`** environment variable is set.  
+By default `BCD_HOME` is set to `/home/bonita/bonita-continuous-delivery` in the controller container.
+* Next the `bcd` command takes a mandatory option called `scenario` through the `-s / --scenario` command line option.  
+This option expects an existing relative or absolute path to a [BCD YAML scenario](scenarios.md) file.  
+The BCD scenario path can also be specified through the **`BCD_SCENARIO`** environment variable, hence allowing to skip the `-s / --scenario` command line option.
+* Finally the BCD CLI expects to find Bonita version-specific dependencies in the **`$BCD_HOME/dependencies`** directory. This directory must contain:
```
Comment (alarreine) [inline md/bcd_cli.md]: Is it better?
 Finally, the BCD CLI expects

File: md/livingapp_build.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+drwxr-xr-x 26 1000 1000 4,0K Mar 29 16:19 web_widgets
+drwxrwxr-x  2 1000 1000 4,0K Mar 29 16:17 xsd
+
+$ grep "<bonita.version>" bonita-vacation-management-example/restAPIExtensions/tahitiRestApiExtension/pom.xml
+        <bonita.version>7.6.3</bonita.version>
+
+$ ls -nh dependencies 
+total 337M
+-rw-r--r-- 1 1000 1000  85M Mar 29 12:00 bonita-la-builder-7.7.0-exec.jar
+-rw-rw-r-- 1 1000 1000 8,7M Mar 29 16:16 bonita-sp-7.6.3-maven-repository.zip
```
Comment (JeremJR) [inline md/livingapp_build.md:88]: even if for test we use this at the moment, shouldn't we put the same version in the documentation?

File: md/aws_sso.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
-# AWS Single Sign-On
+# AWS single sign-on
```
Comment (JeremJR) [inline md/aws_sso.md:1]: maybe that we can also use for this page the `<!--{.h1}-->`, `<!--{.h2}-->` as it's a bit strange to have a toc with only 'Single Sign-On to AWS Using G Suite' ?

File: md/scenarios.md [hunk 8]
```diff
@@ -1,78 +1,96 @@
...
+:::
+
+|Name|Description|Sample|
+|--- |--- |--- |
+|lic_ws_login|Login to Bonitasoft license webservice.|acme|
+|lic_ws_password|Password to Bonitasoft license webservice.|Secr3t|
+|lic_sub_login|Subscription Login provided by Bonitasoft. It has an email address format.|john.doe@acme.com|
+|lic_sub_password|Subscription Password provided by Bonitasoft.|SomePassword|
+|lic_sub_id|Subscription ID provided by Bonitasoft. It's a string of 15 characters.|a0bA0000001B2Cd|
+|lic_type|Type of license in [production|development|qualification|trial].|trial|
```
Comment (JeremJR) [inline md/scenarios.md]: missing `'\'` should be : `[production\|development\|qualification\|trial]`

**Review by JeremJR — CHANGES_REQUESTED** [review]

File: md/bcd_cli.md (hunk 2 above)
Comment (tnguyen1) [reply md/bcd_cli.md:46]: Basically I have used what's in the Click documentation: http://click.pocoo.org/5/commands/#multi-command-chaining

File: md/bcd_cli.md (hunk 3 above)
Comment (tnguyen1) [reply md/bcd_cli.md]: yep looks better :-) https://www.italki.com/question/84025

**Comment by tnguyen1** [comment]
I have integrated your feedback with commit dba3fc7

**Review by JeremJR — APPROVED** [review]

**Pull request merged**